susemanager/bin/mgr-setup

```diff
@@ -812,6 +857,17 @@ do_setup() {
         REPORT_DB_USER="pythia_susemanager"
     fi
 
+    if [ -z "$MANAGER_DB_HOST" ]; then
+        MANAGER_DB_HOST="localhost"
+    fi
+
+    case $MANAGER_DB_HOST in
+        "localhost"|$(hostname -s)|$(hostname -f))
```

Comment:
```suggestion
        "localhost"|$(hostname -s)|$(hostname -f)|"")
```
Empty host is also "localhost". Or we need to force a value already in Setup.pm. This would also work.

Reply:
Thanks for catching this.  I pushed up a fix if you can retest.